apps/src/tutorialExplorer/filterSet.jsx

```diff
@@ -10,12 +10,21 @@ const FilterSet = React.createClass({
     filterGroups: React.PropTypes.array.isRequired,
     onUserInput: React.PropTypes.func.isRequired,
     selection: React.PropTypes.objectOf(React.PropTypes.arrayOf(React.PropTypes.string)).isRequired,
+    mobileLayout: React.PropTypes.bool.isRequired,
     roboticsButtonUrl: React.PropTypes.string
   },
 
+  componentDidMount() {
+    if (this.props.mobileLayout) {
+      $('html, body').animate({
+        scrollTop: $("#filterset").offset().top - $("#filterheader").height() - 6
+      }, 1000);
```

Comment:
Instead of using refs, you could also just do `ReactDOM.findDOMNode(this)` given that filterset is your root node.